docs/reporters/AzureCommentReporter.md

```diff
@@ -42,4 +42,4 @@ Example:
 | Variable                          | Description                                                                      | Default value |
 |-----------------------------------|----------------------------------------------------------------------------------|---------------|
 | AZURE_COMMENT_REPORTER            | Activates/deactivates reporter                                                   | true          |
-| AZURE_COMMENT_REPORTER_LINKS_TYPE | Set to `build` if you want comments links to target Build and not artifacts page | `artifacts`   |
\ No newline at end of file
+| AZURE_COMMENT_REPORTER_LINKS_TYPE | Set to `build` if you want comments links to target Build and not artifacts page | `artifacts`   |
```

Comment:
This is a pre-existing issue, but I believe it should say it should say “comments linking to.”